parsl/data_provider/data_manager.py

```diff
@@ -87,6 +87,6 @@ def stage_out(self, file, executor):
         elif file.scheme == 'globus':
             globus_scheme = _get_globus_scheme(self.dfk, executor)
             stage_out_app = globus_scheme._globus_stage_out_app(executor=executor, dfk=self.dfk)
-            return stage_out_app(inputs=[file])
+            return stage_out_app(app_fu, inputs=[file])
```

Comment:
It doesn't look like the signature for _globus_stage_out matches what it's being called with here.

Reply:
@yadu:

It works when I run it!

stage_out_app is a partially applied decorated form of `_globus_stage_out`. 

The signature of _globus_stage_out is:

```
def _globus_stage_out(scheme, executor, app_fu, inputs=[]):
```

and then `_globus_stage_out_app` partially applies the first two parameters, so that the signature of `stage_out_app` is what remains: `app_fu` and `inputs`.

I think probably when all the staging API changes have settled down, the globus staging provider could be revisited to simplify the code: it's got more complicated as part of adjustments to fit into the upcoming API, but those complications maybe aren't necessary.